barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_format.hpp

```diff
@@ -60,6 +60,7 @@ struct AcirFormat {
     std::vector<poly_triple_<curve::BN254::ScalarField>,
                 ContainerSlabAllocator<poly_triple_<curve::BN254::ScalarField>>>
         constraints;
+    std::vector<mul_quad_<curve::BN254::ScalarField>> quad_constraints;
```

Comment:
Just noting that we may want a `ContainerSlabAllocator` here. But this can be done in a follow-up I feel.

Reply:
I tried to add this.